packages/graphql-api/__tests__/modules/accounts/index.ts

```diff
@@ -0,0 +1,45 @@
+import { AccountsModule } from '../../../src/modules/accounts/index';
+
+const accountsServer = {
+  getServices: () => ({
+    password: {},
+  }),
+};
+
+describe('AccountsModule', () => {
+  describe('withSchemaDefinition', () => {
+    it('should export typeDefs with schema definition', () => {
+      const accountsGraphQL = AccountsModule.forRoot({
+        accountsServer: accountsServer as any,
+        withSchemaDefinition: true,
+      });
+      expect(accountsGraphQL.typeDefs).toMatch(/schema/);
```

Comment:
Could you use `print(accountsGraphQL.typeDefs)` here?

Reply:
👍